MIGRATION_GUIDE.md

```diff
@@ -31,13 +31,67 @@
 
 ### TODO undocumented breaking migration steps
```

Comment:
Need to remove this line

Reply:
We need to cover this point: [Extract scopes from body in mock (#157) ](https://github.com/zalando-incubator/authmosphere/commit/d3961030cf1a5d498b6d960e26f4bb08d3a440a0)

As its breaking an not obvoius.

Reply:
True, this PR is only addressing the first 3 points in the ["Todo list"](https://github.com/zalando-incubator/authmosphere/pull/168/files#diff-697debdb32eda4ce3cf8356fe442602dL32), I will make this explicit with the next commit.